src/sql/workbench/services/objectExplorer/browser/objectExplorerService.ts

```diff
@@ -774,7 +774,7 @@ export class ObjectExplorerService implements IObjectExplorerService {
 		if (expandedState === TreeItemCollapsibleState.Expanded) {
 			await this._serverTreeView?.reveal(expandNode);
 		}
-		return this._serverTreeView?.setExpandedState(expandNode, expandedState);
+		return await this._serverTreeView?.setExpandedState(expandNode, expandedState);
```

Comment:
I think this change may be the responsible one 
(figured this was left unstaged)